src/set_theory/cardinal.lean

```diff
@@ -1023,6 +1023,10 @@ lemma cast_to_nat_of_omega_le {c : cardinal} (h : ω ≤ c) :
   ↑c.to_nat = (0 : cardinal) :=
 by rw [to_nat_apply_of_omega_le h, nat.cast_zero]
 
+lemma to_nat_mono_of_lt_omega {c d : cardinal} (hd : d < ω) (hcd : c ≤ d) :
```

Comment:
This is even an `iff`, right? Can you also prove the `<` version please?

Reply:
Ooh, it's only an `iff` if `c < ω`. But it might still be useful?

Reply:
I've added the `<` version and the `iff` versions.